crt_portal/cts_forms/views.py

```diff
@@ -178,6 +178,74 @@ def post(self, request, id):
         return render(self.request, 'forms/complaint_view/show/index.html', output)
 
 
+class ProFormView(SessionWizardView):
```

Comment:
Since we're not using a multi-stage form, could this be simpler as a regular old `FormView`?

Reply:
Ok, I removed the wizards, there is a bit more dependence on the wizard extras, like steps and management forms in the base template. My instinct is to leave it as is, otherwise, we are pushing the complexity to the templates that would have to account for forms being provided in two different ways.  We can talk about it more later, if you like.